src/components/views/settings/language.tsx

```diff
@@ -24,8 +24,9 @@ export const Language = () => {
       label: langData.name
     })
   );
-
-  const currentLanguage = languages.find(({value}) => value === i18n.language);
+  const currentLanguage = languages.find(
+    ({value}) => value === i18n.language
+  ) || {value: 'en', label: 'English'};
```

Comment:
Rather than duplicating the language object here in the fallback, for tidyness and maintainability can have the fallback re-run the find using the fallback string imported from services/i18n/common ? Then we don't need to remember this if we need to change the languages format in future

Reply:
Something like, put the find in a function then `const currentLanguage = getLanguage(i18n.language) || getLanguage(fallback);`